v20.1/cockroachcloud-cluster-management.md

```diff
@@ -32,11 +28,26 @@ For each node, the page displays the node's `Name`, nested under its region.
 
 From the **Cluster Overview** page, you can connect to your cluster. For more information, see [Connect to Your CockroachCloud Cluster](cockroachcloud-connect-to-your-cluster.html).
 
-## Manage cluster configuration
+## Add or remove nodes from a cluster
+
+You can add or remove nodes from your cluster through the Console.
+
+{{site.data.alerts.callout_info}}
+At this time, you cannot use the Console to scale up a single-node cluster or scale down to a single-node cluster. For single-node cluster changes, contact [Support](https://support.cockroachlabs.com).
+{{site.data.alerts.end}}
+
+### Considerations
+
+- We recommend you add or remove nodes from a cluster when the cluster isn't experiencing heavy traffic.
```

Comment:
Maybe a minor note as to why we recommend this? 
"Adding or removing nodes incurs a non-trivial amount of load on the cluster. Doing so while already experiencing load can result in degraded application performance or longer times for node modifications."

Reply:
Done!